src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -244,6 +244,13 @@ function DatasetOverviewPanel(props) {
               </Accordion>
             </Grid>
           )}
+          <Grid item xs={4}>
+            {renderTextFieldValue(
+              'Custodians Inherit Steward roles on Snapshots',
+              dataset.inheritSteward,
```

Comment:
I don't think this field should ever be rendered as `(empty)` as it is in the screenshot. It's a boolean field that will always have a value as it's non-null in the service layer.

It looks like this may be a side effect of using how `renderTextFieldValue()` uses `TextContent` on a boolean value; the code in `TextContext` evaluates `text` in a way that conflates boolean with undefined.

What happens if you try to convert it to a string

```suggestion
              dataset.inheritSteward?.toString(),
```

Or, maybe handle this in `TextContent`

The fact that it's optional in the generated code is possibly a bug in the model, which could be fixed by making this property required in the openapi yaml.

Reply:
if we make it required do we need to back populate ?